### PR TITLE
chore(java): add image tag to graalvm images

### PR DESCRIPTION
**Pull request by mpeddada1**
Related to #253

### CODE REVIEW

**Comment by mpeddada1** [comment]
Image seems to be tagged:

```
Step #10: 8f229c550c2e: Download complete
Step #6 - "graalvm-build": Removing intermediate container 69e26bb62752
Step #6 - "graalvm-build":  ---> b03e84df8a13
Step #6 - "graalvm-build": Step 8/8 : WORKDIR /workspace
Step #6 - "graalvm-build":  ---> Running in 5056a07a5015
Step #10: 36e653fc8a75: Download complete
Step #6 - "graalvm-build": Removing intermediate container 5056a07a5015
Step #6 - "graalvm-build":  ---> 6bc4ff607c70
Step #6 - "graalvm-build": Successfully built 6bc4ff607c70
Step #6 - "graalvm-build": Successfully tagged gcr.io/cloud-devrel-kokoro-resources/graalvm:v22.2.0
```

**Comment by mpeddada1** [comment]
Next: Apply these changes to cloudbuild.yaml and the Kokoro config files in synthtool.

**Comment by suztomo** [comment]
Is it possible to write the files in a way that we don't have a discrepancy between the installed GraalVM version and the tag name?

For example, passing "22.2" and an (environment) variable `_GRAALVM_IMAGE_TAG ` when building the container and the build should fail if there's a discrepancy.

Another way would be passing "22.2" and an (environment) variable and use the value to determine the base image (`FROM`).

**Comment by suztomo** [comment]
@mpeddada1  I see this is still draft. Anything you're waiting for? May I help something?

File: java/cloudbuild.yaml [hunk 1]
```diff
@@ -133,8 +136,8 @@ images:
   - gcr.io/cloud-devrel-public-resources/java8
   - gcr.io/$PROJECT_ID/java11
   - gcr.io/cloud-devrel-public-resources/java11
-  - gcr.io/$PROJECT_ID/graalvm
-  - gcr.io/cloud-devrel-public-resources/graalvm
+  - gcr.io/$PROJECT_ID/graalvm:${_GRAALVM_IMAGE_TAG}
+  - gcr.io/cloud-devrel-public-resources/graalvm:${_GRAALVM_IMAGE_TAG}
   - gcr.io/$PROJECT_ID/graalvm17
   - gcr.io/cloud-devrel-public-resources/graalvm17
```
Comment (suztomo) [inline java/cloudbuild.yaml:142]: Are they (graalvm17) ok to remain untouched?

File: java/graalvm/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.2.0-b1
+ARG GRAALVM_VERSION
+
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-$GRAALVM_VERSION-b1
```
Comment (suztomo) [inline java/graalvm/Dockerfile:17]:
```suggestion
# If the image is not found with the prefix and suffix, , please find appropriate prefix
# and suffix in ghcr.io/graalvm/graalvm-ce
FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-$GRAALVM_VERSION-b1
```

"-b1" seems ad-hoc and I expect it change in next releases.

File: java/cloudbuild.yaml (hunk 1 above)
Comment (mpeddada1) [reply java/cloudbuild.yaml]: Fixed. Thank you

**Comment by mpeddada1** [comment]
> @mpeddada1 I see this is still draft. Anything you're waiting for? May I help something?

Thanks @suztomo. Was just waiting for the cloud build job to finish running before marking the PR as ready.

**Comment by suztomo** [comment]
My github account (when used with suggestion) didn't work for CLA. Let's me see whether I can approve the CLA in my personal account.

Screen Shot 2022-11-28 at 11 12 41 AM

**Review by suztomo — APPROVED** [review]

File: java/graalvm/Dockerfile [hunk 3]
```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.2.0-b1
+ARG GRAALVM_VERSION
+
+# If the image is not found with the prefix and suffix, , please find appropriate prefix
```
Comment (suztomo) [inline java/graalvm/Dockerfile:17]:
```suggestion
# If the image is not found with the prefix and suffix, please find appropriate prefix
```

**Comment by mpeddada1** [comment]
Confirmation:

```
Step #6 - "graalvm-build": Successfully built 0292438a8461
Step #6 - "graalvm-build": Successfully tagged gcr.io/cloud-devrel-kokoro-resources/graalvm:22.2.0
```

**Pull request merged**